lms/templates/main.html

```diff
@@ -58,7 +61,10 @@
     })(this);
   </script>
 
-  <script type="text/javascript" src="/jsi18n/"></script>
+  % if not enable_require_js:
```

Comment:
This probably deserves a comment. Is i18n.js included in the bundle if ```enable_require_js``` is true?